applications/StructuralMechanicsApplication/custom_constitutive/generic_finite_strain_isotropic_plasticity.h

```diff
@@ -115,10 +115,7 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) GenericFiniteStrainIsotropicP
           mPlasticDissipation(rOther.mPlasticDissipation),
           mThreshold(rOther.mThreshold),
           mPlasticDeformationGradient(rOther.mPlasticDeformationGradient),
-          mNonConvPlasticDissipation(rOther.mNonConvPlasticDissipation),
-          mNonConvThreshold(rOther.mNonConvThreshold),
-          mNonConvPlasticDeformationGradient(rOther.mNonConvPlasticDeformationGradient),
-          mUniaxialStress(rOther.mUniaxialStress)

```

Comment:
I suppose that they are in order.. otherwise in Win will have problems to compile :)

Reply:
They are

Reply:
Travis makes sure of this ;)